.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing to azure-rest-api-specs
+First, thank you for contributing to our Azure specs repository! Specs are the basis for generating Azure SDKs in multiple different languages, Azure CLIs for interacting with the services, and can provide documentation for these services.
+
+## Basics
+In the Azure Developer Experience workflow, you are at Step 3:
+
+[API Design Review](https://github.com/Azure/adx-documentation-pr#begin-api-design-review) -> [Engage with ADX team](https://github.com/Azure/adx-documentation-pr/blob/master/README.md#engage-with-adx-team) -> _**[Swagger specification](https://github.com/Azure/adx-documentation-pr#create-swagger-specification)**_ -> [SDKs](https://github.com/Azure/adx-documentation-pr#sdks) -> [CLIs](https://github.com/Azure/adx-documentation-pr#clis)
+
+If you're a spec author looking for information about all of of the repositories and steps in the pipeline, go back to the [adx-documentation-pr](https://github.com/Azure/adx-documentation-pr) repository.
+
+## Table of Contents
+[Before starting](#before-starting)
+- [Onboarding](#onboarding)
+- [Github basics](#github-basics)
+- [Code of Conduct](#code-of-conduct)
+
+[Making changes](#making-changes)
+- [Documentation](#documentation)
+- [Filenames and folder structure](#filenames-and-folder-structure)
+- [Tools for writing Swagger](#tools-for-writing-swagger)
+- [Tools for validating Swagger](#tools-for-validating-swagger)
+
+[Submitting a PR](#submitting-a-pr)
+
+[Review process](#review-process)
+- [SLA](#sla)
+- [Review Criteria page](../documentation/review-criteria.md)
+
+## Before starting
+
+### Onboarding
+Before cloning this repository, please make sure you have started in our [documentation repository adx-documentation-pr](https://github.com/Azure/adx-documentation-pr).
```

Comment:
Really good point, that causes some confusion for some teams when I direct them to adx-documentation-pr and they ping me since they don't have access. I'll add that before merging